### ISSUE TITLE
Default new reminders have an immediate notification

### ISSUE THREAD

**Issue by neomodern** [jonasbleyl/recurrence#52]
If you create a new reminder and write in just a title only and save (quickly adding a reminder in a rush) then you get an immediate notification. It should default to maybe 15 minutes or an hour ahead.

**Comment by jonasbleyl** [comment]
Yeah, this would be better. There should probably be a section in the settings to set defaults for all of the reminder settings though.
